templates/default/esxi-ks.cfg.erb

```diff
@@ -0,0 +1,10 @@
+# Accept the VMware End User License Agreement
+vmaccepteula
+# Configure a static IP Address on the first network adapter
+network --bootproto=dhcp --device=vmnic0
+# Set the root password for the DCUI and Tech Support Mode
+rootpw Ubuntu!!
```

Comment:
Why not make this configurable? (and if you choose to do so, don't forget to update the README :smile:)